### ISSUE TITLE
zarr.tree() on h5py File in notebook renders with wrong icon for root node and datasets

### ISSUE THREAD

**Issue by alimanfoo** [zarr-developers/zarr-python#262]
#### Problem description

``zarr.tree()`` is designed to work with either zarr Group or h5py File. When passed an h5py File the icons rendered in the jupyter notebook are wrong for the root node (should be black folder icon like groups) and for datasets (should be table icon). E.g.:

![image](https://user-images.githubusercontent.com/703554/39860591-c239aa5e-5435-11e8-8096-9f93afbb7d02.png)

#### Version and installation information

* Value of ``zarr.__version__``: 2.2.0
* Version of Python interpreter: 3.6

**Comment by jakirkham** [comment]
Yeah, this must be some underlying representation of [jsTree]( https://www.jstree.com/ ) showing up in the representation. It's strange that it only happens with HDF5 files. Maybe something about how we determine what a Dataset is is not working. Perhaps this issue is simpler than it looks.

**Comment by jakirkham** [comment]
Is this still an issue with recent releases? Wondering if this is gone after moving to ipytree?

**Comment by maxrjones** [comment]
Closing as part of V2-era issue triage. The `zarr.tree()` rendering has been completely rewritten in V3 and no longer uses jsTree/ipytree. The V3 tree implementation works on `zarr.Group` objects only, not h5py files.

Please reopen if this is still relevant.
